openlibrary/plugins/openlibrary/js/automatic.js

```diff
@@ -11,11 +11,13 @@ export default function($) {
     });
 
     // wmd editor
-    $('textarea.markdown').wmd({
-        helpLink: '/help/markdown',
-        helpHoverTitle: 'Formatting Help',
-        helpTarget: '_new'
-    });
+    $markdownTextArea = $('textarea.markdown');
```

Comment:
Const this since we're adding a new thing.

```suggestion
    const $markdownTextarea = $('textarea.markdown');
```

Reply:
done